server/src/main/java/org/apache/iotdb/db/query/reader/series/SeriesReader.java

```diff
@@ -266,6 +270,10 @@ Statistics currentChunkStatistics() {
     return firstChunkMetadata.getStatistics();
   }
 
+  boolean currentChunkModified() {
```

Comment:
if firstChunkMetadata is null, throw exception

Reply:
fixed